app/Application.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * bool $approved
+ * enum $gamemode ['osu', 'taiko', 'mania', 'catch']
+ * enum $verdict? ['fail', 'pass']
+ * int $cycle_id
+ * int $user_id
+ * int $feedback_author_id
+ * string $feedback
+ */
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Application extends Model
+{
+    protected $casts = [
+        'approved' => 'boolean',
+    ];
+
+    protected $fillable = [
+        'approved',
+        'cycle_id',
+        'feedback',
+        'feedback_author_id',
```

Comment:
each application can only receive a single user's feedback? is this temporary?

Reply:
it's meant to be PM feedback